guide-us-english.tex

```diff
@@ -305,10 +283,17 @@ \subsection{Scaling the SoS}
 
 \subsection{The Executive Action Team}
 The Scrum of Scrums for the entire agile organization is called the
-\textbf{Executive Action Team (EAT)}. The EAT is the final stop for
+\textbf{Executive Action Team (EAT)}. This leadership team creates an agile bubble
+in the organization with its own guidelines and procedures that integrates effectively 
+with any part of the organization that is not agile. It owns the agile ecosystem, 
+implements the Scrum values, and assures that 
+Scrum roles are created and supported.
+
+The EAT is the final stop for
 impediments that cannot be removed by the SoS's that feed it. Therefore, it
 must be comprised of individuals who are empowered, politically and
-financially, to remove them. The function of the EAT is to coordinate
+financially, to implement and change the structure of the agile ecosystem. 
```

Comment:
I don't think this adds anything, it is a bit redundant, and we are talking about impediments.  The focus should be on "remove them".  There is no final stop to implementing change and continuous improvement.